config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/PolicyLoader.java

```diff
@@ -46,6 +46,7 @@ public void load(Bundle bundle, Element element) {
         final String policyType = getSingleChildElementTextContent(policyDetails, POLICY_TYPE);
         final Map<String, Object> policyDetailProperties = mapPropertiesElements(getSingleChildElement(policyDetails, PROPERTIES, true), PROPERTIES);
         final String policyTag = (String) policyDetailProperties.get(PROPERTY_TAG);
+        final boolean hasRouting = BooleanUtils.toBoolean((String) policyDetailProperties.get(PROPERTY_HAS_ROUTING));
```

Comment:
is this value becomes true for all routing assertions?

Reply:
Yes. If the policy contains any Routing assertion, it will be true.

Reply:
Please add JUnits to cover this

Reply:
@RaghuReddy done